### PR TITLE
Replaced dashing with a sprinting feature

### PR DESCRIPTION
**Pull request by Rikir-M**
Fixed https://github.com/Glowstick0017/Little-Plane-Project/issues/62

I set the duration during sprinting to 1. Please tell me if it looks too fast or slow. I also removed cooldown since I believe it won't be needed during sprinting.

### CODE REVIEW

**Comment by Glowstick0017** [comment]
if possible, can you make it so that it's a constant speed while holding shift and not a bunch of small boosts.

**Comment by Rikir-M** [comment]
Ahh you mean kinda like the feature when you change the speed with the command? Sure, let me try!

**Comment by Rikir-M** [comment]
@Glowstick0017 hello, I've changed the logic for sprinting. Please review and tell me if there's something you'd want me to fix.

File: engine/Controls.js [hunk 1]
```diff
@@ -78,10 +80,8 @@ function moveRotateAndSprint(dx, dy, sprinting , keyPressedFlag) {
 
     // Update the position of plane based on the direction of movement and sprint status , only if any of the keys is pressed
     if (keyPressedFlag) {
-        posX += dx * speed * 10 + 30 * dx * t;
```
Comment (Glowstick0017) [inline engine/Controls.js:85]: removing the `* t` in these two lines seems to have made the speed of the plane by default incredibly fast.

Comment (Rikir-M) [reply engine/Controls.js:85]: Hello! I've fixed the bug by adding t back.

**Comment by Glowstick0017** [comment]
#67

**Pull request closed without merging**